### PR TITLE
Minor improvements to the README

### PR DESCRIPTION
**Pull request by sir4ur0n**
* Fix typos
* Fix 4th element list display
* Add the shell hint on code blocks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,50 +124,50 @@ for help about a specific subcommand.
...
+  ```shell
   $ cargo doc --no-deps
   ```
 2. Open the file `target/doc/nickel/index.html` in your browser.
 
 ### Examples
 
 You can find examples in
-[`examples`](./examples). Note
+[`/examples`](./examples) directory. Note
```
Comment (gkaracha) [inline README.md:163]: I think it was  better before, but even if it is to be changed I think we need the dot here:
```suggestion
[`./examples`](./examples) directory. Note
```

Comment (sir4ur0n) [reply README.md:163]: I find it weird-looking to be both an inline code (backticks) AND a link:
image

I'd go with:
* either don't make it inline code, i.e. only write [examples](./examples)
* or make it explicit it refers to the directory (in which case I agree with your suggestion of adding the dot), i.e. [`./examples`](./examples)

Comment (gkaracha) [reply README.md]: Yeah, I see. It's fine either way for me; let's see what @yannham prefers :slightly_smiling_face:

**Review by yannham — CHANGES_REQUESTED** [review]
Thanks for the fix! I'm not sure about the `shell` hint, which I think is for shell scripts, not sessions (it randomly colors subcommand like `test` in `cargo test` because it is a shell command while `cargo doc` is not colored, it colors the `let` in the REPL, and so on). I think the `console` hint behaves more like a session but I've never really used it, maybe should we give it a try?

File: README.md [hunk 2]
```diff
@@ -123,50 +124,50 @@ for help about a specific subcommand.
...
+  ```shell
   $ cargo doc --no-deps
   ```
 2. Open the file `target/doc/nickel/index.html` in your browser.
 
 ### Examples
 
 You can find examples in
-[`examples`](./examples). Note
+[`./examples`](./examples) directory. Note
```
Comment (yannham) [inline README.md:163]:
```suggestion
the [`./examples`](./examples) directory. Note
```

**Comment by sir4ur0n** [comment]
> Thanks for the fix! I'm not sure about the `shell` hint, which I think is for shell scripts, not sessions (it randomly colors subcommand like `test` in `cargo test` because it is a shell command while `cargo doc` is not colored, it colors the `let` in the REPL, and so on). I think the `console` hint behaves more like a session but I've never really used it, maybe should we give it a try?

@yannham done in last commit, you can see the result in https://github.com/tweag/nickel/blob/e38f53a60d14d9100c2e8328ba8a49f10c1fbdbe/README.md

**Review by yannham — APPROVED** [review]

**Pull request merged**